src/tests/account/test_dual_eth_account.cairo

```diff
@@ -4,7 +4,9 @@ use openzeppelin::account::eth_account::dual_eth_account::{
 use openzeppelin::account::eth_account::interface::{
     EthAccountABIDispatcherTrait, EthAccountABIDispatcher
 };
-use openzeppelin::account::utils::secp256k1::{Secp256k1PointPartialEq, Secp256k1PointSerde};
+use openzeppelin::account::utils::secp256k1::{
+    DebugSecp256k1Point, Secp256k1PointPartialEq, Secp256k1PointSerde
```

Comment:
does `assert_eq!` require the Debug impl?

Reply:
Yes